lib/plugins/aws/provider.js

```diff
@@ -924,6 +924,7 @@ class AwsProvider {
                 metrics: { type: 'boolean' },
                 useProviderTags: { const: true },
                 disableDefaultEndpoint: { type: 'boolean' },
+                shouldStartNameWithService: { type: 'boolean' },
```

Comment:
I think it should be `const: true` instead as setting it to `false` does not make sense, so the only reasonable value is `true`

Reply:
Totally agree. I've already updated it. Thanks!